tutorials/vue/vue-d3.md

```diff
@@ -0,0 +1,148 @@
+[D3.js](https://d3js.org/) is a great library for visualizing data and displaying it in your projects. This library gives you the building blocks to make very aesthetically pleasing charts in projects. To use it
+with your Vue 3 projects, you can download a libray called `vue-d3-charts`
+which is built upon `D3.js`. `vue-d3-charts` does not support vue2 and so
+you will need to import `D3.js`.
+
+As of this writing, development for vue-d3-charts is currently frozen as the head maintainer does not have the time
```

Comment:
Ditto, not worth it to explain vue-d3-charts at the beginning if the example doesn't use it